client/src/components/AddNewTask.js

```diff
@@ -105,7 +113,7 @@ const TimeElement = styled.p`
   grid-row: 4/5;
 `;
 
-function DetailCard() {
+function AddNewTask() {
```

Comment:
what exactly does this component? the category buttons have no function/onclick etc.. I am confused because you also have edit card.

Reply:
if you want to create a new task you click on the plus button and then you see the addnewtaskcard.:)

Reply:
okay but then the buttons need to have active etc as well 😉